### PR TITLE
docs: fix skywalking-logger typo

### PR DESCRIPTION
**Pull request by Gallardot**
### Description

Fix skywalking-logger typo

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ title: skywalking-logger
 
 ## 描述
 
-`skywalking-logger` 是一个插件，可将 Access Log 数据通过 `HTTP` 推送到 SkyWalking OAP 服务器。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，并依赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/) 。
+`skywalking-logger` 是一个可将 Access Log 数据通过 `HTTP` 推送到 `SkyWalking OAP` 服务器的插件。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，这有赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/)的支持 。
```
Comment (zaunist) [inline docs/zh/latest/plugins/skywalking-logger.md:26]:
```suggestion
`skywalking-logger` 是一个可将 Access Log 数据通过 `HTTP` 推送到 `SkyWalking OAP` 服务器的插件。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，这依赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/)的支持 。
```

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ title: skywalking-logger
...
 
-这将提供将 Access Log 数据作为 JSON 对象发送到 SkyWalking OAP 服务器的功能。
+本插件提供将 Access Log 数据作为JSON对象发送到 `SkyWalking OAP` 服务器的功能。
```
Comment (zaunist) [inline docs/zh/latest/plugins/skywalking-logger.md:28]:
```suggestion
本插件提供将 Access Log 数据作为 JSON 对象发送到 `SkyWalking OAP` 服务器的功能。
```

Comment (zaunist) [reply docs/zh/latest/plugins/skywalking-logger.md:28]: We need a black space between Chinese and English, you could refer to https://github.com/sparanoid/chinese-copywriting-guidelines.

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 3]
```diff
@@ -87,7 +87,7 @@ hello, world
 
 ## 禁用插件
 
-在插件配置中删除相应的 json 配置以禁用 skywalking-logger。APISIX 插件是热重载的，因此无需重新启动 APISIX ：
+在插件配置中删除相应的 json 配置就可以禁用 skywalking-logger。APISIX 插件是热重载的，因此无需重新启动 APISIX ：
```
Comment (hf400159) [inline docs/zh/latest/plugins/skywalking-logger.md:90]:
```suggestion
在插件配置中删除相应的 JSON 配置就可以禁用 `skywalking-logger` 插件。由于APISIX 插件支持热加载的，因此无需重新启动 APISIX：
```

**Review by hf400159 — COMMENTED** [review]
“JSON”  should be uppercase.

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 4]
```diff
@@ -23,9 +23,9 @@ title: skywalking-logger
 
 ## 描述
 
-`skywalking-logger` 是一个插件，可将 Access Log 数据通过 `HTTP` 推送到 SkyWalking OAP 服务器。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，并依赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/) 。
+`skywalking-logger` 是一个可将 Access Log 数据通过 `HTTP` 推送到 `SkyWalking OAP` 服务器的插件。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，这依赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/)的支持 。
```
Comment (yzeng25) [inline docs/zh/latest/plugins/skywalking-logger.md:26]:
```suggestion
`skywalking-logger` 是一个可将 Access Log 数据通过 `HTTP` 推送到 `SkyWalking OAP` 服务器的插件。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，这依赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/)的支持。
```

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 5]
```diff
@@ -87,7 +87,7 @@ hello, world
 
 ## 禁用插件
 
-在插件配置中删除相应的 json 配置以禁用 skywalking-logger。APISIX 插件是热重载的，因此无需重新启动 APISIX ：
+在插件配置中删除相应的 JSON 配置就可以禁用 `skywalking-logger` 插件。由于APISIX 插件支持热加载的，因此无需重新启动 APISIX：
```
Comment (yzeng25) [inline docs/zh/latest/plugins/skywalking-logger.md:90]:
```suggestion
在插件配置中删除相应的 JSON 配置就可以禁用 `skywalking-logger` 插件。由于APISIX 插件支持热加载，因此无需重新启动 APISIX：
```

**Comment by Gallardot** [comment]
Thank you for working with you on this PR. PTAL. 
@yzeng25 @zaunist @hf400159

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 6]
```diff
@@ -73,11 +73,11 @@ HTTP/1.1 200 OK
 hello, world
 ```
 
-完成上述步骤后，在可以 SkyWalking UI 查看到相关日志。
+完成上述步骤后，可以在`SkyWalking UI` 查看到相关日志。
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/skywalking-logger.md:76]:
```suggestion
完成上述步骤后，可以在 `SkyWalking UI` 查看到相关日志。
```

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 7]
```diff
@@ -87,7 +87,7 @@ hello, world
 
 ## 禁用插件
 
-在插件配置中删除相应的 json 配置以禁用 skywalking-logger。APISIX 插件是热重载的，因此无需重新启动 APISIX ：
+在插件配置中删除相应的 JSON 配置就可以禁用 `skywalking-logger` 插件。由于APISIX 插件支持热加载，因此无需重新启动 APISIX ：
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/skywalking-logger.md:90]:
```suggestion
在插件配置中删除相应的 JSON 配置就可以禁用 `skywalking-logger` 插件。由于 Apache APISIX 插件支持热加载，因此无需重新启动 Apache APISIX：
```

**Review by juzhiyuan — COMMENTED** [review]
Thanks! Please also update this PR's checklist.

**Comment by Gallardot** [comment]
> Thanks! Please also update this PR's checklist.

Done. PTAL.

**Comment by yzeng25** [comment]
Hi @Gallardot , please add a brief description(i.e. Fix skywalking-logger typo) and check the 4th box of the checklist.

**Review by juzhiyuan — APPROVED** [review]
LGTM, please update the checklist.

**Review by yzeng25 — APPROVED** [review]
LGTM. Thanks @Gallardot !

**Review by zaunist — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**